db/schema.rb

```diff
@@ -0,0 +1,62 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_08_08_222002) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "balance_items", force: :cascade do |t|
+    t.bigint "item_id", null: false
+    t.bigint "balance_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["balance_id"], name: "index_balance_items_on_balance_id"
+    t.index ["item_id"], name: "index_balance_items_on_item_id"
+  end
+
+  create_table "balances", force: :cascade do |t|
+    t.string "name"
+    t.string "icon"
+    t.bigint "author_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.float "total_amount", default: 0.0
```

Comment:
- [OPTIONAL] Rather than approximate data type, I would suggest you use decimal which stores the exact values. Right now I can't pass any decimal value. 
![image](https://github.com/Ouail-01/Budget-app/assets/91486791/f39fa193-504e-43c6-8caa-9a1f1571ecc3)